tests/acceptance/pageObjects/filesPage.js

```diff
@@ -66,6 +66,21 @@ module.exports = {
       }
       return null
     },
+    /**
+     * Check if the breadcrumb element is visible or not
+     *
+     * @returns {Promise<module.exports.commands>}
```

Comment:
I think this is still not right. It returns some nightwatch object in `return this`
I think you can just leave it as return Promise

Reply:
@dpakach it's done.